assets/pages/support-landing/supportLanding.jsx

```diff
@@ -54,7 +54,7 @@ const content = (
       <WhySupport />
       <ReadyToSupport ctaUrl="#" />
       <PatronsEventsContainer />
-      <Footer />
+      <Footer disclaimer privacyPolicy />
```

Comment:
Where are `disclaimer` and `privacyPolicy` coming from? I assume they're passed through `store` or something but I don't see them being evaluated or imported. This also applies to https://github.com/guardian/support-frontend/pull/512/files#diff-47b1e022b2fe0aa95f3c0eaa6c1ca9f0R49